lib/Doctrine/ORM/EntityManager.php

```diff
@@ -813,44 +816,6 @@ public function initializeObject($obj)
     }
 
     /**
-     * Factory method to create EntityManager instances.
-     *
-     * @param mixed         $conn         An array with the connection parameters or an existing Connection instance.
-     * @param Configuration $config       The Configuration instance to use.
-     * @param EventManager  $eventManager The EventManager instance to use.
-     *
-     * @return EntityManager The created EntityManager.
-     *
-     * @throws \InvalidArgumentException
-     * @throws ORMException
-     */
-    public static function create($conn, Configuration $config, EventManager $eventManager = null)
```

Comment:
@mcspronko BC breaks are forbidden in minor versions. Semver means that any BC break in the public API requires bumping the major version